src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/Account/Manage/Account.Manage.TwoFactorAuthentication.cshtml

```diff
@@ -1,57 +1,72 @@
 @page
+@using Microsoft.AspNetCore.Http.Features
 @model TwoFactorAuthenticationModel
 @{
     ViewData["Title"] = "Two-factor authentication (2FA)";
     ViewData["ActivePage"] = ManageNavPages.TwoFactorAuthentication;
 }
 
-<partial name="_StatusMessage" for="StatusMessage" />
+<partial name="_StatusMessage" model="Model.StatusMessage" />
 <h4>@ViewData["Title"]</h4>
-@if (Model.Is2faEnabled)
-{
-    if (Model.RecoveryCodesLeft == 0)
+@{ 
+    var consentFeature = HttpContext.Features.Get<ITrackingConsentFeature>();
```

Comment:
@blowdart I don't remember exactly where the consent stuff landed, can you review how you expect/want it to behave after it gets scaffolded

Reply:
I think @DamianEdwards wanted it pulled out

Reply:
Ok so this shouldn't be in the scaffolding bits, and probably can get removed from the UI on our side since its been removed from the template app

Reply:
ok reverted